src/main/java/com/hotels/bdp/circustrain/bigquery/extraction/container/DeleteTableAction.java

```diff
@@ -31,8 +32,12 @@ public DeleteTableAction(Table table) {
 
   @Override
   public void run() {
-    table.delete();
-    log.debug("Deleted table '{}'", table.getTableId());
+    try {
+      table.delete();
+      log.debug("Deleted table '{}'", table.getTableId());
+    } catch (BigQueryException e) {
```

Comment:
Also needs to be mentioned in changelog - if an error occurs deleting an intermediate table during cleanup this will no longer fail the replication but now will just log an error and continue.